README.md

```diff
@@ -1,2 +1,146 @@
+<p align="justify">
 # Star-Wars-BB9
 Srishti 2020
+
+<p align="center">
+  <img width="450" height="550" src="https://github.com/Kritin02/Star-Wars-BB9/blob/master/Images%20and%20Videos/Images/Interior.jpeg?raw=true">
+</p>
+
+## Abstract
+The droid should replicate BB-8 from the Star Wars franchise complete with a rotating body but a stationary head.The movement of the droid can be controlled by a companion app. The body can rotate freely and can be moved in all direction freely using the app.The head is established by using strong Neodymium magnets and ball casters.* 
+
+
+## Motivation
+After watching the BB-8 from Star Wars, we were interested in its working and mechanism. As the newbies, out of curiosity we choose to work on the same.*
+
+## Components Used
+### Mechanical:
+
+|       Component      |                         Size                        | Quantity |
+|:--------------------:|:---------------------------------------------------:|:--------:|
+| Neobium magnets      |                      Dia- 4mm                       |    18    |
+|     Nut and Bolt     |                       4mm dia                       |    20    |
+|     L - Brackets     |                       Standard                      |     2    |
+|     Motor Mount      |                      Dia-3.6cm                      |     2    |
+|     Castor Wheel     |                        Small                        |     3    |
+|       Plywood 1      |                  Dia-24cm, t=0.5cm                  |     8    |
+|       Plywood 2      |                  Dia-11.5cm,t-0.5cm                 |     4    |
+|      Globe           |                      Dia- 30cm                      |     1    |  
+|      Pipe            |                  Dia-2cm,L-7cm                      |     1    |
+|      Wheels          |                        Dia-7.2cm                    |     2    |
+
+### Electronics
+* 1 X Arduino Uno
+* 3 X L298(Motor Driver)
+* 2 X 60rpm DC motors
+* 2 X 12V Li-polymer Batteries
+* Jumpers
+
+## Flowchart
+<img src="https://i.imgur.com/RjfrX5S.png">
+
+## Mechanical of the Design
```

Comment:
Remove this images and add your solid works design these are copied images from internet.

Reply:
Sir this is not copied image I have made the flowchart and uploaded it into my Imgur account to use here